flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/catalog/SchemaTranslator.java

```diff
@@ -113,19 +114,42 @@ public static ProducingResult createProducingResult(
             ResolvedSchema inputSchema,
             AbstractDataType<?> targetDataType) {
         final List<String> inputFieldNames = inputSchema.getColumnNames();
+        final List<String> inputFieldNamesNormalized =
+                inputFieldNames.stream()
+                        .map(n -> n.toUpperCase(Locale.ROOT))
```

Comment:
I think it's better to use `toLowerCase` since usally, most of the fields names would be lowercase, wdyt?

Reply:
I don't have a strong opinion. Just thought that upper case is more common to show that we don't care often in SQL you have field names like `SELECT USER_ID FROM EMP;`, no?

Reply:
Me neither, just in my experience I'm more used to see `SELECT user_id FROM emp`.